apps/framework-docs/src/pages/building/data-models/capture-data.mdx

```diff
@@ -65,37 +71,33 @@ Following is some example code for testing the API endpoints for a Data Model ca
   and let us know.
 </Callout>
 
-To facilitate the ingestion process, Moose automatically generates a TypeScript SDK for every data model defined in your project. The SDK provides instant type safety and schema validation, offering real-time feedback on the data you intend to send to Moose during development.
+To facilitate the ingestion process, Moose can generate automatically a TypeScript SDK for every data model
+defined in your project. The SDK provides instant type safety and schema validation, offering real-time
+feedback on the data you intend to send to Moose during development.
 
-While Moose also provides HTTP endpoints for data ingestion (see above), we recommend utilizing the SDK for more robust data integrity enforcement and reduced runtime errors. We're actively expanding support to include more programming languages and welcome your input on which ones to prioritize.
+While Moose also provides HTTP endpoints for data ingestion (see above), we recommend utilizing the SDK for
+more robust data integrity enforcement and reduced runtime errors. We're actively expanding support to
+include more programming languages and welcome your input on which ones to prioritize.
 
 ### Getting Started with the SDK
 
-Ensure your Moose application’s development server is running. If it’s not, run `moose dev` and/or [refer to the documentation on starting the development server](https://docs.moosejs.com/getting-started/new-project).
+From within your moose project you can generate the SDK for your project by running the following command:
 
-Then follow these steps to prepare your development environment for SDK usage:
-
-### Linking the SDK Globally
-
-Before using the SDK to send data to Moose from your other applications, it needs to be linked to your global Node.js modules.
-
-1. _Navigate to the SDK Directory_: From your Moose project’s root directory, execute the following command `cd .moose/<YOUR_PROJECT_NAME>-sdk`
-2. _Execute the Link Command_: Use the appropriate command for your node package manager to create a global symlink. Run the following command in your terminal from the SDK directory: for npm: `npm link`; for pnpm `pnpm link -g`.
-
-### Integrating the SDK with Your Application
-
-Once you've linked the SDK to your global node modules, integrate it into your applications as follows:
-
-1. _Navigate to Your Application_: Go to the root directory of the application where you want to use the SDK, ensuring the `package.json` file is present.
-2. _Link the SDK_: Use the command corresponding to your package manager to create a symlink in your application's node_modules that points to the SDK. Replace `<YOUR_MOOSE_PROJECT_NAME>` with the name of your Moose project: for npm: `npm link <YOUR_MOOSE_PROJECT_NAME>-sdk`; for pnpm `pnpm link -g <YOUR_MOOSE_PROJECT_NAME>-sdk`.
+```bash
```

Comment:
Looks like other places do this so it indicates this is something to run in the terminal, and they could click a button to copy the text. If we do make it copy-able, we probably want to remove the dollar sign so it doesn't get copied.

`txt filename="Terminal" copy`

Reply:
Good call - thank you